pkg/services/govmomi/vcenter/clone.go

```diff
@@ -338,7 +339,7 @@ func getDiskLocators(disks object.VirtualDeviceList, datastoreRef types.ManagedO
 	return diskLocators
 }
 
-func getDiskSpec(ctx *context.VMContext, devices object.VirtualDeviceList) ([]types.BaseVirtualDeviceConfigSpec, error) {
+func getDiskSpec(_ context.Context, vmCtx *capvcontext.VMContext, devices object.VirtualDeviceList) ([]types.BaseVirtualDeviceConfigSpec, error) {
```

Comment:
if there's no context or logger using in getDiskSpec, there's no need to add new parameter here.

Reply:
Removed the context for `getDiskSpec` func.